packages/rich-text-react-renderer/src/__test__/__snapshots__/index.test.tsx.snap

```diff
@@ -193,14 +193,14 @@ Array [
 `;
 
 exports[`documentToReactComponents renders nodes with passed custom node renderer 1`] = `
-Array [
+<Document>
```

Comment:
Could you add one more test case to verify that we return Array if Document Node Renderer is not passed? This will ensure that we are not introducing any breaking changes with this feature.

Reply:
Good catch!